tensorboard/plugins/histogram/histograms_demo.py

```diff
@@ -24,7 +24,11 @@
 LOGDIR = "/tmp/histograms_demo"
 
 
-def run(k):
+def run(k, step):
+    """
+    Arguments:
+        k: a float in the range [0, 1] that affects the histogram values written by the run.
```

Comment:
Minor, but could we document `step` here too?  If we're documenting any argument, I think it's best to just document all of them to avoid being inconsistent between the docs and the signature.